content/faq/loginradius-support.md

```diff
@@ -43,7 +43,7 @@ To access the LoginRadius Support section, login to your [LoginRadius Dashboard]
 </div>
 <br/>
 
-Apart from this, you can also check out **Need help?** section available at the bottom of getting started page as displayed in the below screen:
```

Comment:
@nickc95 
This is related to `How to access the LoginRadius Support section?` so mentioning Need help section would be ok here as all the links under this are related to the support section.

Reply:
Also, the blur part is not consistent with the existing images in docs.

Reply:
Updated blur filter